libs/core/src/lib/menu/menu-item/menu-item.component.ts

```diff
@@ -84,8 +106,8 @@ export class MenuItemComponent implements DefaultMenuItem, OnInit, OnChanges, Af
     menuItemTitle: MenuTitleDirective;
 
     /** @hidden Reference to the Menu Item interactive element */
-    @ContentChild(MenuInteractiveDirective)
-    menuInteractive: MenuInteractiveDirective;
+    @ContentChild(forwardRef(() => MenuInteractiveComponent))
```

Comment:
Have you figured out why direct usage of component is failing the tests?

Reply:
Oh, it does not fail anymore, it was because of circular dependencies, on a logical level, not on a file system level, now those are fixed and it works without them. I will remove this `forwardRef` since it is not needed anymore